package.json

```diff
@@ -128,5 +128,9 @@
       "tslint --fix -t verbose",
       "git add"
     ]
+  },
+  "volta": {
```

Comment:
pinned versions of tools for react-use development purposes
this is using volta-cli